### DISCUSSION TITLE
How to reset local supabase storage?

### DISCUSSION THREAD

**Discussion by techwizzdom** [supabase/supabase#18517] [Questions]
Something got messed up with my local Supabase Storage - every time I click on an already existing bucket, or create a new one, I get:
```
Failed to retrieve folder contents from "bucket_id": An error has occurred: signal is aborted without reason
```
In network, I can see 404 for:
```
http://localhost:54323/project/default/storage/buckets/undefined/projects/default/custom-hostname
```
I assume the problem is in `undefined` here. As a result, I can't view files in the storage at `http://localhost:54323/project/default/storage/buckets/bucket_id`.

I also get this error message on `supabase start`: `service not healthy: [supabase_storage_sparroww]`.

**My question is: how to reset the local environment to the initial state?**

I tried:
1. Deleting `storage.buckets`, `storage.objects`, `storage.migrations`, and recreating them,
2. Complelety destroying all the Docker resources with `docker system prune -a`,

But I still get the same error message when I do `supabase start` and it doesn't work properly.

**Comment by JerryKwan** [comment]
@techwizzdom 
According to the code in https://github.com/supabase/cli/blob/v1.168.1/internal/start/start.go#L764
you can try to stop and re-create supabase_db_secondbrain, supabase_storage_secondbrain, please try with the following commandd
```
supabase stop
docke container rm supabase_db_secondbrain
docker container rm supabase_storage_secondbrain
docker volume rm supabase_db_secondbrain
docker volume rm supabase_storage_secondbrain
supabase start
```
